figure.html

```diff
@@ -398,7 +398,136 @@
 </div> <!-- end #line-block -->
 
 <li>
-<p class="first">Click <span class="bold">Edit Figure Legend</span> to add a legend to selected images.</p>
+
+<p class="first">On the <span class="bold">Labels</span> tab click <span class="bold">Draw ROIs</span> to add ROI shapes to selected panes.</p>
+
+<p>In the ROIs window, draw rectangle, line, arrow and ellipse ROIs on the panel.</p>
+
+<p>Line colour and thickness can be selected using the drop-downs.</p>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<img alt="../images/figureROI.jpg" class="align-center" src="images/figureROI.jpg" style="width: 100%;" />
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+</li>
+
+<li>
+
+<p class="first">ROI shapes can be edited or moved after adding, and copied and pasted using the <span class="bold">Edit</span> menu or keyboard shortcuts.</p>
+
+<p>Click <span class="bold">OK</span> to save the shapes.</p>
+
+<p>Copy shapes in the Draw ROIs window, close the window and select panels in the main Figure drawing.</p>
+
+<p>Use keyboard shortcuts to paste the ROIs directly onto the selected panels.</p>
+
+<div class="note">
+<p class="note-title">Note</p>
+<p class="note-text">Pasted shapes are not scaled, so if pasted onto panels with a different size or zoom level, they may not fit on the panel.</p>
```

Comment:
I would say this a little differently: "Shapes will be pasted at the same coordinates as on the source image. If the target image is smaller or zoomed in to a different region, then shapes may not be visible. In this case a warning message will be displayed".